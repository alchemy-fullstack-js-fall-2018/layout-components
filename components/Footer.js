import Component from './Component';
import styles from './Footer.css';

export default class Footer extends Component {
  render() {
    const content = this.props.content;

    return `
      <footer class="${styles.footer}">
          ${content}
      </footer>`;
  }
}
