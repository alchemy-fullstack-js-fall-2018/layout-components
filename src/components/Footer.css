.footer{
  display: flex;
  justify-content: center;
  background-color: grey;
  width: 100%;
}



@media screen and (min-width: 700px) {

  .footer {
    display: flex;
    justify-content: center;
    background-color: grey;
    width: 100%;
    height: 25px;
    padding: 20px;
    font-weight: bolder;
    font-size: 24px;
    position: absolute;
    bottom: 0;
    left: 0;
  }
}
