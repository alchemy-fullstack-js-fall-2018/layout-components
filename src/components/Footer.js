import Component from './Component';
import styles from './Footer.css';

export default class Footer extends Component {
  render() {
    return `
      <footer class="${styles.footer}">
        <a>&copy;Mack BG</a>
      </footer>
    `;
  }

}
