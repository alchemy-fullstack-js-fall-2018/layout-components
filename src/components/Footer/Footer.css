.footer {
  background-color: silver;
  color: black;
}
