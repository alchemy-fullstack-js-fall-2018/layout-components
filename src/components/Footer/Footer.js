import Component from '../Component/Component';

import styles from './Footer.css';
export default class Footer extends Component {

  render() {

    return `
      <footer class="${styles.footer}">
        <p>Made to entertain - 2018</p>
      </footer>
    `;
  }


}
