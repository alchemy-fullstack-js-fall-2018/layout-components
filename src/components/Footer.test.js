import Footer from './Footer';

describe('Footer component', () => {
  it('is a footer', () => {
    const footer = new Footer();
    expect(footer.render()).toMatchSnapshot();
  });

});
