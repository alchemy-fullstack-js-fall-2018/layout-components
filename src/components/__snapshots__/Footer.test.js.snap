// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component is a footer 1`] = `
"
      <footer class=\\"footer\\">
        <a>&copy;Mack BG</a>
      </footer>
    "
`;
